llvm/test/CodeGen/AMDGPU/fptrunc.f16.ll

```diff
@@ -196,7 +196,7 @@ define amdgpu_kernel void @fptrunc_f32_to_f16(
     ptr addrspace(1) %a) {
 entry:
   %a.val = load float, ptr addrspace(1) %a
-  %r.val = fptrunc float %a.val to half
+  %r.val = fptrunc afn float %a.val to half
```

Comment:
Can you duplicate the functions to have a safe and unsafe version, and remove the -enable-unsafe-fp-math command line arguments